server/graphql/v1/mutations/orders.js

```diff
@@ -845,7 +845,7 @@ export async function refundTransaction(_, args, req) {
   //   b. Host Collective receiving the donation
   //   c. Site Admin
```

Comment:
@xdamman can you perhaps change these comments which are now outdated?

Reply:
Ah sorry, only seeing your comment after merging. Shall I do another PR or just push another commit for that small change in master?

Reply:
New PR: https://github.com/opencollective/opencollective-api/pull/2662